### PR TITLE
Feature - `disable_sbpf_v1_execution` for tests

### PR DESCRIPTION
**Pull request by Lichtso**
#### Problem
The feature `reject_callx_r10` can not be cleaned up because some tests rely on it (see #2414).

These tests require a feature to toggle the verification success of a program. Currently we use very specific properties of the programs to trigger the rejection logic. This however is cumbersome as these programs need to be manually crafted, so a more general approach would be nice.

#### Summary of Changes
Adds two dummy features `disable_sbpf_v1_execution` and `reenable_sbpf_v1_execution` for tests, which is can not be activated on any cluster (would require rekeying) and replaces `reject_callx_r10` with these features. Two features are required because we enable all features by default. Thus, any single test feature which breaks programs on purpose would break all unit tests and local test clusters.

### CODE REVIEW

File: runtime/src/bank/tests.rs [hunk 1]
```diff
@@ -12063,7 +12062,7 @@ fn test_feature_activation_loaded_programs_epoch_transition() {
     let feature_account_balance =
         std::cmp::max(genesis_config.rent.minimum_balance(Feature::size_of()), 1);
     bank.store_account(
-        &feature_set::reject_callx_r10::id(),
+        &feature_set::disable_fees_sysvar::id(),
```
Comment (pgarg66) [inline runtime/src/bank/tests.rs:12065]: why was this necessary?

Comment (Lichtso) [reply runtime/src/bank/tests.rs:12065]: Because the `reject_callx_r10` is cleaned up, so it has no more influence. Had to find another RBPF related feature. But for this test the programs don't need to match the feature, so any RBPF related feature works.

**Review by pgarg66 — APPROVED** [review]
